### DISCUSSION TITLE
Fixed Navbar, Bottom NavBar, Drawer & Footer Example: How to set the footer at the bottom of the page?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#3042] [Q&A]
Hi

Can I please get help?

Q1. I need the footer to be at the bottom of the page and fixed.
Q2. Is this the best approach to try and achieve what is here? i.e a fixed nav & fixed footer. For the bottom nav I was just testing and that works well, it would be nice if the footer could behave the same way. 

Thanks in advance.

Here is the link to tailwind play..

https://play.tailwindcss.com/pqOXFOuHbx?size=1408x1362

![image](https://github.com/saadeghi/daisyui/assets/5799306/ed8cb853-7818-4c7e-b1df-55962e6ac97c)

**Comment by MrVibe** [comment]
you can either use position sticky or position fixed 

position-sticky bottom-0 with overflow visible

**Comment by saadeghi** [comment]
I don't see why you would need bottom nav and footer/navbar on the same page 😅 Too much navigation.

https://play.tailwindcss.com/1XLDbGJQmZ?size=1408x1362

**Comment by Mashiane** [comment]
Thank you so much, this is exactly what I needed. An option to use a footer or use a bottom nav. Now I know.

Option 1:

![image](https://github.com/saadeghi/daisyui/assets/5799306/e31e7efe-ca01-4c46-a04a-b641c074e084)

Option 2:

![image](https://github.com/saadeghi/daisyui/assets/5799306/3db2e7f0-3bee-491f-85a4-17b315a948d2)

https://play.tailwindcss.com/sRsvsOPviQ?size=1408x1362

https://play.tailwindcss.com/8BcftfbwKk?size=1408x1362
